### ISSUE TITLE
[BUG] Goblin crown redirect (visual)

### ISSUE THREAD

**Issue by gakons** [sunflower-land/sunflower-land#1067]
## Summary of Bug

Visual bug when redirecting through goblin crown.

## Steps to Reproduce

Click on the goblin crown in inventory

## Screenshots

![Снимок](https://user-images.githubusercontent.com/87533524/172795461-17b38264-8246-4915-95aa-52b1e0ef5a45.JPG)
![bug](https://user-images.githubusercontent.com/87533524/172800061-46c671b1-96b2-40cd-b665-c91bad7e522b.gif)

**Device Information :**

- Device: [PC]
- OS: [Windows 10]
- Browser/App [Chrome]
- Version [102]

**Comment by future2back** [comment]
So what is the function of this NFT?
Documentation is not clear.

**Comment by j923sneaks** [comment]
forwarding initial findings:

https://discord.com/channels/880987707214544966/929861853201436713/984340972160811018


https://user-images.githubusercontent.com/89294757/172871906-0c09a866-806b-4a9c-86dc-4f81b19fe2fa.mp4



TLDR: i was the one who brought back the goblin couple (fix of prev bug) but this current bug behavior was not encountered during dev (on local instance). Found some leads but still have not figured out the root cause

**Comment by adamhannigan** [comment]
Thanks for raising this. The team has identified an issue that was related to the build/web hosting.

This has been fixed and will be released shortly 🤗
